package.json

```diff
@@ -224,7 +224,7 @@
     "@opam/decoders-yojson": "*"
   },
   "resolutions": {
-    "reason-jsonrpc": "github:bryphe/reason-jsonrpc#95f2427",
+    "reason-jsonrpc": "onivim/reason-jsonrpc#ebadf6d",
```

Comment:
Seems like this got lost somewhere, therefore re-doing it so that we can get the trace logging back.